File: week1/cpp/sum.cpp
#include <cstdio>
#include <iostream>

int sumOfTwoDigits(int a, int b) {
    return a+b;
}

int main() {
    int a;
    a = 2;
    int b = 1;
    printf("Result is: %d", sumOfTwoDigits(a, b));

    std::cin >> a;
    std::cin >> b;
    std::cout << a + b << std::endl;
    return 0;
}

File: week1/golang/sum.go
package main

import "fmt"

func main() {

	var a, b int
	a = 2
	b = 1

	fmt.Println("Golang Result is", sumOfTwoDigits(a, b))
}

func sumOfTwoDigits(a, b int) int {
	return a + b
}
